scripts/index-demo-page.js

```diff
@@ -8,14 +8,23 @@
 
   var getInputValue = function(el) {
     if (el.type === 'url' || el.type === 'text') {
-      return el.value;
+      return encodeURIComponent(el.value);
     } else if (el.type === 'checkbox') {
       return el.checked;
     }
 
     console.warn('unhandled input type ' + el.type);
   };
 
+  var setInputValue = function(el, value) {
```

Comment:
Moved this to a function so that set and get are right next to one another, should make it harder to forget.